packages/devtools/lib/src/framework/framework.dart

```diff
@@ -307,65 +318,6 @@ class Framework {
     }
   }
 
-  void showInfo({String message, String title, List<CoreElement> children}) {
```

Comment:
Could these still exist, and just delegate to `messageManager`?

Reply:
done. The message type (info vs warning) will be specified in the message itself so we only need one method here: `showMessage`. 

`showError` will be handled separately as it has some custom logic.